Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -119,7 +119,7 @@ void Jit64::lXXx(UGeckoInstruction inst)
 	// ... maybe the throttle one already do that :p
 	// TODO: We shouldn't use a debug read here.  It should be possible to get
 	// the following instructions out of the JIT state.
-	if (SConfig::GetInstance().bSkipIdle &&
```

Comment:
Mostly referring to the "dynamically enable/disable Idle Skipping" part, since its either on or off (or with this PR, always on). Not really sure about the conversation between ector and mb2 tho.